### PR TITLE
Describe CPU and Memory resources.

### PR DESCRIPTION
**Pull request by erictune**

### CODE REVIEW

File: docs/user-guide/compute-resources.md [hunk 1]
```diff
@@ -40,6 +40,32 @@ about pod resource requests/limits.  A *pod resource request/limit* for a partic
...
+### Meaning of CPU
+Limits and requests for `cpu` are measured in cpus.  
+One cpu, in Kubernetes, is equivalent to:
+
+- 1 AWS vCPU
+- 1 GCP Core
+- 1 Azure vCore
+- 1 *Hyperthread* on a bare-metal Intel processor with Hyperthreading
+
+Fractional requests are allowed.  A container with `spec.container[].resources.requests.cpu` of `0.5` will
```
Comment (vishh) [inline docs/user-guide/compute-resources.md]: Would it be helpful to explicitly state that Kubernetes does not normalize across CPU types with different performance characteristics?

File: docs/user-guide/compute-resources.md [hunk 2]
```diff
@@ -40,6 +40,32 @@ about pod resource requests/limits.  A *pod resource request/limit* for a partic
...
+to mean the same thing when talking about Kubernetes).  A request with a decimal point, like `0.1` is converted to
+`100m` by the API, and precision finer than `1m` is not allowed.  For this reason, the form `100m` may be preferred.
+
+CPU is always requested as an absolute quantity, never as a relative quantity; 0.1 is the same amount of cpu on a single
+core, dual core, or 48 core machine.
+
+# Meaning of Memory
+
+Limits and requests for `memory` are measured in bytes.
+Memory can be expressed a plain integer or as fixed-point integers with one of these SI suffixes (E, P, T, G, M, K, m)
```
Comment (vishh) [inline docs/user-guide/compute-resources.md]: `m` is not valid for memory right?

File: docs/user-guide/compute-resources.md (hunk 1 above)
Comment (erictune) [inline docs/user-guide/compute-resources.md]: I think that is implied from the first paragraph.

File: docs/user-guide/compute-resources.md (hunk 2 above)
Comment (erictune) [inline docs/user-guide/compute-resources.md]: You can specify `m`, which seems like a bug.

Comment (erictune) [inline docs/user-guide/compute-resources.md]: Changed docs to reflect how it should work

Comment (erictune) [inline docs/user-guide/compute-resources.md]: filed https://github.com/kubernetes/kubernetes/issues/28741

**Comment by vishh** [comment]
LGTM

**Comment by MHBauer** [comment]
This is good information for users to have.

**Pull request merged**